hw10/server.js

```diff
@@ -0,0 +1,39 @@
+const http = require('http');
+const fs = require('fs');
+
+const server = http.createServer((req, res) => {
+  const regImg = /\.jpg$|\.png$|\.jpeg$|\.gif$|\.svg$/;
```

Comment:
it is not optimized expression , please use next one
`/\.(jpg|png|jpeg|gif|svg)$/`